Networking/Networking/Model/Product/ProductSubscription.swift

```diff
@@ -83,8 +98,7 @@ public struct ProductSubscription: Decodable, Equatable, GeneratedFakeable, Gene
             .init(key: CodingKeys.signUpFee.rawValue, value: signUpFee),
             .init(key: CodingKeys.trialLength.rawValue, value: trialLength),
             .init(key: CodingKeys.trialPeriod.rawValue, value: trialPeriod.rawValue),
-            .init(key: CodingKeys.oneTimeShipping.rawValue, value: oneTimeShipping ? Constants.yes : Constants.no),
-            .init(key: CodingKeys.paymentSyncDate.rawValue, value: paymentSyncDate),
```

Comment:
Are we removing this from metadata intentionally? I suppose since we don't update the field, it makes sense not to encode it. Maybe we should leave a comment to say why the field is not encoded.

Reply:
Good question. 

Yes, I stopped encoding as we don't support editing payment sync data in the app. Also, we will have to encode these values into String/JSON format based on the subscription settings. 

Added a comment to explain this in https://github.com/woocommerce/woocommerce-ios/pull/11312/commits/74f36dd1842a6cb2fde2031be8c87ca035d7cb17